<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowInfo</class>
 <widget class="QWidget" name="ShowInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инфлрмация о сотруднике</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>721</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>label_12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>label_11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>label_15</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>label_9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>label_7</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Отдел</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Транспорт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Имейл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Мобильный телефон</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>321</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>label_6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>171</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>label_16</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>131</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Вход/Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>161</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>label_18</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>171</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>label_19</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>721</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>261</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Отработана сегодня:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
